src/components/EndreInntektVsaPensjon/EndreInntektVsaPensjon.tsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react'
+import { FormattedMessage, useIntl } from 'react-intl'
+
+import { PencilIcon, TrashIcon } from '@navikt/aksel-icons'
+import { Button, Label, Modal, TextField } from '@navikt/ds-react'
+
+import { validateInntektInput } from '../EndreInntekt/utils'
+import { useAppDispatch, useAppSelector } from '@/state/hooks'
+import { selectCurrentSimulation } from '@/state/userInput/selectors'
+import { userInputActions } from '@/state/userInput/userInputReducer'
+import { formatWithoutDecimal } from '@/utils/currency'
+
+import styles from './EndreInntektVsaPensjon.module.scss'
+
+interface Props {
+  temporaryStartAlder?: string
+}
+
+// TODO logger
+export const EndreInntektVsaPensjon: React.FC<Props> = ({
```

Comment:
Jeg regner med at denne kan brukes både for inntekt vsa. gradert pensjon og for inntekt vsa. hel pensjon? (Bare så vi ikke får noen navngivingsutfordringer :)

Reply:
Det ser ut på skissen at inntekt ved siden av gradert pensjon skal bestå av et enkelt input felt direkte i skjemaet. Så, jeg tror ikke den skal få sin egen komponent. Jeg holder denne kommentaren i bakhodet når jeg begynner på det.